lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php

```diff
@@ -0,0 +1 @@
+<?php print $image; ?>
```

Comment:
This is just for the template in the module, that doesn't get used cause it's overridden in the theme ;)